setup.py

```diff
@@ -23,16 +23,18 @@ def read_version(package):
     'clickclick>=1.2,<21',
     'jsonschema>=2.5.1,<5',
     'PyYAML>=5.1,<7',
-    'requests>=2.19.1,<3',
+    'requests>=2.27,<3',
```

Comment:
Why do we need to bump to 2.27?

Reply:
Because of a sub-dependency which is shared by both `requests` and `starlette`. We'll be able to drop the `requests` dependency once we move security into the middleware stack though, since we'll only need the `AsyncSecurityHandlerFactory` then.